deploy/crds/istio_v1alpha2_istiocontrolplane_cr.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: install.istio.io/v1alpha2
 kind: IstioControlPlane
 metadata:
+  namespace: istio-operator
```

Comment:
does the namespace indicate the control plane is deployed to Istio-operator namespace?

Reply:
Yes, currently in-cluster controller will be also installed into the same namespace.